### DISCUSSION TITLE
Termux start rish with arguments

### DISCUSSION THREAD

**Discussion by TechnicJelle** [RikkaApps/Shizuku#233] [Q&A] [answered]
Hi! I'm using rish in Termux to access the ADB shell and I was wondering if it was possible to start rish with arguments to make it, for example, start `top` immediately, without having to type in top after having started rish. With that I could make a shortcut on my home screen to start up a task manager that actually works.

**Comment by RikkaW** [comment, marked as answer]
https://github.com/RikkaApps/Shizuku-API/tree/master/rish#usage

`rish -c 'top'`

**Reply by TechnicJelle** [reply]
Thanks! And sorry for not seeing that myself 😅

**Reply by TechnicJelle** [reply]
It works beautifully!

**Reply by 9724164533** [reply]
Od

**Reply by hwachhmchhmay-coder** [reply]
> Thanks! And sorry for not seeing that myself 😅

Thanks
